mlflow/pyfunc/__init__.py

```diff
@@ -2429,9 +2447,23 @@ def predict(model_input: List[str]) -> List[str]:
             extra_pip_requirements=extra_pip_requirements,
             model_config=model_config,
             streamable=streamable,
+            model_code_path=model_code_path,
```

Comment:
Do we need this in the first `if` case as well? Similar to model_config, streamable, etc?

Reply:
```
first_argument_set = {
        "loader_module": loader_module,
        "data_path": data_path,
    }
    second_argument_set = {
        "artifacts": artifacts,
        "python_model": python_model,
    }
```

I believe that we aren't supporting the functionality from the `first_argument_set` yet, so I didn't add it to that if statement. Once we support custom `loader_module`s with logging pyfunc as code, I was planning to add it there. wdyt?